stable/consul/templates/consul-test.yaml

```diff
@@ -4,21 +4,20 @@ metadata:
   name: "{{ .Release.Name }}-ui-test-{{ randAlphaNum 5 | lower }}"
   annotations:
     "helm.sh/hook": test-success
-    "pod.beta.kubernetes.io/init-containers": '[
-      {
-        "name": "test-framework",
-        "image": "dduportal/bats:0.4.0",
-        "command": ["bash", "-c", "
-          set -ex\n
-          # copy bats to tools dir\n
-          cp -R /usr/local/libexec/ /tools/bats/\n
-        "],
-        "volumeMounts": [
-          {"name": "tools", "mountPath": "/tools"}
-        ]
-      }
-    ]'
 spec:
+  initContainers:
+    - name: test-framework
+      image: dduportal/bats:0.4.0
+      command:
+        - "ba$h"
+        - "-c"
+        - |
+          set -ex
+          # copy bats to tools dir
+          cp -R /usr/local/libexec/ /tools/bats/
+      volumeMounts:
+      - mountPath: /tools
```

Comment:
inconsistent indentation for lists here, this line should probably indented another 2 spaces